components/Comments/ReportPost.tsx

```diff
@@ -15,6 +16,11 @@ interface Props {
 
 export const ReportPost = (props:Props) => {
 
+  const { mutate:sendEmail }  = trpc.sendEmail.send.useMutation({ 
+      onSuccess: () => { toast.success('Report sent')},
+      onError: () => {  toast.error('Oops, something went wrong, please send us a message on discord https://github.com/codu-code/codu');  }
+  })
```

Comment:
I have an issue here.  The user will alsways recieve the success toast. Any error with sending the email is handled by nodemailer, and I dont know how to throw it here on the front end. I experimented with try catch errors, but could not get it to work.

Reply:
I created a route and moved the sendEmail function to the backend. Hopefully the email will now work.